docs/ref/django-admin.txt

```diff
@@ -1279,7 +1279,7 @@ fly.
 For example, taking advantage of GitHub's feature to expose repositories as
 zip files, you can use a URL like::
 
-    django-admin startapp --template=https://github.com/githubuser/django-app-template/archive/master.zip myapp
+    django-admin startapp --template=https://github.com/githubuser/django-app-template/archive/main.zip myapp
```

Comment:
"main" is the default branch name for new github repos, so this more accurately reflects what users would see if they created their own template repo